rust/processor/migrations/2024-05-21-221101_add_royalty_v1/up.sql

```diff
@@ -0,0 +1,24 @@
+-- Your SQL goes here
+CREATE TABLE token_royalty (
+    transaction_version BIGINT NOT NULL,
+    token_data_id VARCHAR(66) NOT NULL,
+    payee_address VARCHAR(66) NOT NULL,
+    royalty_points_numerator NUMERIC NOT NULL,
+    royalty_points_denominator NUMERIC NOT NULL,
+    token_standard TEXT NOT NULL,
+    inserted_at TIMESTAMP NOT NULL DEFAULT NOW(),
+    PRIMARY KEY (transaction_version, token_data_id) 
+);
+
+CREATE TABLE current_token_royalty (
```

Comment:
do we need any indices here?

Reply:
wait...where is collection_id. we need that to get the join key.

Reply:
it's only going to join with `current_token_datas.token_data_id`

Reply:
`token_data_id` is already part of the PK so no additional index needed

Reply:
I'd suggest not adding unnecessary columns as part of PK. 256 bits hash is used everywhere and assumes no collision.

Reply:
This won't work for v2 though. Let's rename the table to v1 then.